src/main/java/core/basesyntax/service/parsefileinfo/FruitParser.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.parsefileinfo;
+
+import core.basesyntax.service.functionalityexpansion.ActivityType;
+
+public class FruitParser {
+    private static final int ACTIVITY_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+    private static final String SEPARATOR = ",";
+
+    public FruitInfo parseActivity(String activityString) {
```

Comment:
you are not parsing activity here, you return transaction, but this can be ommited since class name already defined the context that it is a transaction that we are working with
```suggestion
    public FruitTransaction parse(String activityString) {
```